### PR TITLE
DEVDOCS-3807: Update checkout-sdk

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3807](https://jira.bigcommerce.com/browse/DEVDOCS-3807)

## What changed?
added a section describing how to update the checkout-sdk

### CODE REVIEW

**Review by slsriehl — DISMISSED** [review]
I made some changes. I'm not at all sure what's going on here. From the ticket it seems like one of the bundled dependencies of @bigcommerce/checkout-sdk is maybe being manually updated, but it's not straightforward to me that checkout-js is actually a dependency of checkout-sdk. 

I'm not sure devdocs is the thing that needs to change to address this issue. If it is, we need more context to provide to users.

@traci707 @hatertron3000

**Comment by traci707** [comment]
Thank you @hatertron3000 , I will work with the checkout team.

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 1]
```diff
@@ -53,6 +51,34 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
...
+# Clone the repo
+git clone https://github.com/<your-user-name>/checkout-js checkout-js
+
+cd checkout-js
+
+# Install dependencies with a clean slate
+npm ci
+```
+
+2. Open your store's `package.json` file in your text editor.
```
Comment (animesh1987) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:69]: You can also go to the terminal and run `npm install @bigcommerce/checkout-sdk@<latest/version number>`

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 2]
```diff
@@ -53,6 +51,34 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
...
+3. Update the value for "@bigcommerce/checkout-sdk". Enter the minimum version for the gateway to work.
```
Comment (bc-peng) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:70]: Think it's worth mentioning we provide checkout-sdk version information on [this page](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md).

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 3]
```diff
@@ -42,8 +40,8 @@ With the Checkout JS SDK, you can build a [custom checkout presentation](https:/
 
 The Checkout JS SDK allows you to present the checkout process to the shopper in any way you like and, through front-end technologies and external web services, augment the checkout with extra information.
 
-However, the Checkout JS SDK does not allow you to change the checkout's underpinnings   - you must still conform to the model of the Checkout API underpinning the SDK to complete a Checkout and create an Order in BigCommerce.
-The SDK does not allow you to implement custom payment, shipping, or tax calculation providers into the checkout. Instead, you can use the configured providers for these services on a given store via our Checkout API.
+However, the Checkout JS SDK does not allow you to change the checkout's underpinnings -- you must still conform to the model of the Checkout API underpinning the SDK to complete a Checkout and create an Order in BigCommerce.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:43]:
```suggestion
However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.
```

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 4]
```diff
@@ -53,6 +51,17 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
 ## Where can I get the SDK?
 The Checkout JS SDK and associated documentation is available from the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js)
 
+## How can I update the SDK?
+The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.
+
+1. Update the SDK by either running `npm install @bigcommerce/checkout-sdk@<latest-version-numer>` in your terminal, or
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:57]:
```suggestion
1. Update the SDK in one of the two following ways:

* Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in your terminal. 

```

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 5]
```diff
@@ -53,6 +51,17 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
...
+update the version number in your checkout's package.json file, then run npm install.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:58]: * Update the version number in your checkout implementation's package.json file, then run `npm install` in your terminal.

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 6]
```diff
@@ -53,6 +51,17 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
...
+ 
+<!-- checkout: info -->
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:60]:
```suggestion
<!-- theme: info -->
```

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 7]
```diff
@@ -53,6 +51,17 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
...
+> Note: See [Changelog.md](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md) for checkout-sdk version information.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:61]:
```suggestion
> #### Note 
> See the checkout-sdk-js repo's [changelog.md](https://github.com/bigcommerce/checkout-sdk-js/blob/master/CHANGELOG.md) for checkout-sdk version information.
```

**Comment by slsriehl** [comment]
@traci707 suggestions for clarity, and a minor typo. Let me know!

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 8]
```diff
@@ -54,11 +54,15 @@ The Checkout JS SDK and associated documentation is available from the [BigComme
 ## How can I update the SDK?
 The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.
 
-1. Update the SDK by either running `npm install @bigcommerce/checkout-sdk@<latest-version-numer>` in your terminal, or
-update the version number in your checkout's package.json file, then run npm install.
+1. In your terminal, navigate to the project root of your customer checkout implementation. You can update the SDK in one of the two following ways:
+
+* Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in the terminal.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:59]:
```suggestion
* Run `npm install @bigcommerce/checkout-sdk@<latest-version-number>` in the terminal; or 
```

File: docs/api-docs/cart-and-checkout/checkout-sdk.md [hunk 9]
```diff
@@ -42,8 +40,8 @@ With the Checkout JS SDK, you can build a [custom checkout presentation](https:/
 
 The Checkout JS SDK allows you to present the checkout process to the shopper in any way you like and, through front-end technologies and external web services, augment the checkout with extra information.
 
-However, the Checkout JS SDK does not allow you to change the checkout's underpinnings   - you must still conform to the model of the Checkout API underpinning the SDK to complete a Checkout and create an Order in BigCommerce.
-The SDK does not allow you to implement custom payment, shipping, or tax calculation providers into the checkout. Instead, you can use the configured providers for these services on a given store via our Checkout API.
+However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.
```
Comment (slsriehl) [inline docs/api-docs/cart-and-checkout/checkout-sdk.md:43]:
```suggestion
However, the Checkout JS SDK does not allow you to change the checkout's underpinnings. You must still conform to the SDK's model of the Checkout API to complete a checkout and create an order in BigCommerce.

```

add a carriage return between paragraphs.

**Review by animesh1987 — APPROVED** [review]

**Review by bc-tgomez — APPROVED** [review]
LGTM 🎉

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**